.gitmodules

```diff
@@ -5,5 +5,5 @@
 [submodule "external/LibAPR"]
 	path = external/LibAPR
 	url = https://github.com/AdaptiveParticles/LibAPR.git
-	branch = develop_joel
+	branch = tmp_install_windows_joel
```

Comment:
should remember to make this master once we've pushed through the merge in LibAPR

Reply:
Yeah i'll update this to master.

Reply:
I'm a bit confused on when this is used vs when you just add the commit update via git add? I have done both.